apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -26,11 +26,11 @@ import {
   SigningServiceResponse,
 } from '@island.is/dokobit-signing'
 import { InjectQueue, QueueService } from '@island.is/message-queue'
+// import { MessageType } from '@island.is/judicial-system/message'
```

Comment:
Remove redundant code?

Reply:
These imports are temporarily commented out along with some queuing code that is not working because the queuing config on dev is incomplete. We don't want to remove the code because it will be uncommented as soon as Andes has fixed the config.